Makefile

```diff
@@ -111,7 +111,7 @@ nodejs:
 	$(call run-cmake-release, -DBUILD_NODEJS=TRUE)
 
 python:
-	$(call run-cmake-release, -DBUILD_PYTHON=TRUE)
+	$(call run-cmake-debug, -DBUILD_PYTHON=TRUE)
```

Comment:
Shouldn't we make release for python test by default?

Reply:
Yes. I need to stop leaving these changes in.